meson_options.txt

```diff
@@ -20,3 +20,4 @@ option('use_sys_zlib', type: 'boolean', value: false)
 option('use_sys_lz4', type: 'boolean', value: false)
 option('use_sys_xxhash', type: 'boolean', value: false)
 option('use_sys_openssl', type: 'boolean', value: false)
+option('use_libuv', type: 'boolean', value: true)
```

Comment:
Maybe `use_sys_libuv`?

Reply:
There is no bundled alternative. If it's set to false, it doesn't use libuv at all.